core/domain/email_manager.py

```diff
@@ -934,3 +938,90 @@ def send_test_email_for_bulk_emails(tester_id, email_subject, email_body):
     _send_email(
         tester_id, tester_id, feconf.BULK_EMAIL_INTENT_TEST,
         email_subject, email_body, tester_email, sender_name=tester_name)
+
+
+def send_mail_to_onboard_new_reviewers(user_id, category):
+    """Sends an email to users who have attained high scores by contributing
+    suggestions.
+
+    Args:
+        user_id: str. The ID of the user who is being offered to become a
+            reviewer.
+        category: str. The category that the user is being offered to review.
+    """
+
+    email_subject = 'Invitation to review suggestions'
+
+    email_body_template = (
+        'Hi %s,<br>'
+        'You have been actively contributing quality suggestions for oppia '
+        'content in %s. We would like to have you review other contributors '
```

Comment:
`In recognition of your contributions, we would like to have you as one of our reviewers. As a reviewer, you would be responsible for regularly reviewing the suggestions in the same category and ensuring that the accepted content meets the quality bar.` Hm, looks like we need a guideline for reviewing content to which we can refer people to. @seanlip what do you think?

Reply:
Done

Reply:
Yeah I think we need to have some guidelines.

Reply:
Ok, I'll file an issue for this.